apps/site/components/MDX/Image/index.tsx

```diff
@@ -2,14 +2,18 @@ import type { ImageProps } from 'next/image';
 import Image from 'next/image';
 import type { FC } from 'react';
 
-const MDXImage: FC<ImageProps> = ({ width, height, alt, ...props }) => {
+import { isSvgImage } from '@/util/imageUtils';
+
+const MDXImage: FC<ImageProps> = ({ width, height, alt, src, ...props }) => {
```

Comment:
Just a small tweak since we'll be setting this property anyway, but LGTM
```suggestion
const isUnoptimizedImage = isSvgImage(src.toString())
```